src/configuration.ts

```diff
@@ -984,7 +984,7 @@ export function defaultSourceMapPathOverrides(webRoot: string): { [key: string]:
     'webpack://?:*/*': `${webRoot}/*`,
     'webpack:///([a-z]):/(.+)': '$1:/$2',
     'meteor://💻app/*': `${webRoot}/*`,
-    'turbopack://[project]/*': '${workspaceFolder}/*',
+    'turbopack:///[project]/*': '${workspaceFolder}/*',
```

Comment:
Should this be optional for back-compat with old versions?

```suggestion
    'turbopack:///?[project]/*': '${workspaceFolder}/*',
```